apiary.apib

```diff
@@ -681,6 +681,38 @@ IN PROGRESS
 
 + Response 204
 
+## Work Order Task Collection [/accounts/{accountId}/workorders/{workOrderId}/tasks{?page,pagesize,orderby}]
+
++ Parameters
+    + accountId: 1 (required, number) - #INT ID of the account in the form of an integer
```

Comment:
I know this is nit picking (spelling?) but some of these descriptions seem to be in sentence form and some seem to be just statements. It would be nice to see them all standardized to be sentence form but I'm not sure if that's even necessary. @brianhyder Do we have any sort of standards around apiary docs?